submodules/radios/aprs.py

```diff
@@ -0,0 +1,150 @@
+import logging
+from functools import partial
+from time import time, sleep
+
+from . import Radio
+from core import ThreadHandler
+
+from serial import Serial
+
+
+class APRS(Radio):
+    def __init__(self, config):
+        """
+        Assumes APRS is in low power mode on start. Sets up class fields.
+        :param config: the config dictionary loaded from config_default.yml
+        """
+
+        self.config = config
+        self.modules = dict()
+
+        self.logger = logging.getLogger("APRS")
+        self.last_telem_time = time()
+        self.last_message_time = time()
+
+        self.serial = None
+        self.processes = {
+            "listen_thread": ThreadHandler(
+                target=partial(self.listen),
+                name="aprs-listen",
+                parent_logger=self.logger,
+            )
+        }
+
+    def start(self):
+        """
+        Opens the APRS serial port and starts the listening thread.
+        Assumes enough power is present therefore the tty port exists.
+        """
+        self.serial = Serial(self.config["aprs"]["serial_port"], 19200)
+        self.processes["listen_thread"].start()
```

Comment:
instead of specifically referencing the `listen_thread` thread, iterate through `self.processes` and start each value
This will make it easier to add more processes later on and is consistent with The Framework
```
for i in self.processes:
  self.processes[i].start()
```